saml2/saml2-service-provider/src/test/java/org/springframework/security/saml2/provider/service/authentication/OpenSaml4AuthenticationProviderTests.java

```diff
@@ -729,6 +735,77 @@ public void authenticateWhenCustomResponseValidatorThenUses() {
 		verify(validator).convert(any(OpenSaml4AuthenticationProvider.ResponseToken.class));
 	}
 
+	@Test
+	public void setsOnlyParentStatusCodeOnResultDescription() {
```

Comment:
Will you please name your tests like the following: `authenticateWhenXXXThenYYY` where `XXX` is the situation under test and `YYY` is the expected result?